docs/main/post-install.md

```diff
@@ -6,4 +6,17 @@ If your router does not support setting the DNS server, you can [use Pi-hole's b
 
 As a last resort, you can always manually set each device to use Pi-hole as their DNS server.
 
+## Make your Pi-hole host use Pi-hole
+
+As described [here](https://docs.pi-hole.net/main/prerequisites/#ip-addressing) your Pi-hole host device should use a static IP address. Pi-hole won't be used by the host automatically after installation. However, you can make the host use Pi-hole as upstream DNS server.
```

Comment:
```suggestion
Pi-hole will not be used by the host automatically after installation. To have the host resolve though Pi-hole and your configured blocking lists, you can make the host use Pi-hole as upstream DNS server:
```

I don't think mentioning static addressing really relates to this topic. 

The other changes are merely stylistic.